examples/view-alias/index.mjs

```diff
@@ -0,0 +1,54 @@
+import { loadStdlib, util } from '@reach-sh/stdlib';
+import * as backend from './build/index.main.mjs';
+
+const stdlib = loadStdlib(process.env);
+
+const startingBalance = stdlib.parseCurrency(100);
+
+const [accAlice, accBob] =
+  await stdlib.newTestAccounts(2, startingBalance);
+
+const ctcAlice = accAlice.contract(backend);
+
```

Comment:
You could add a test that the ABI (on one of the connectors) is what you expect it to be.